code/modules/mob/living/carbon/human/human.dm

```diff
@@ -949,10 +949,24 @@
 	return (ishuman(target) && !(target.mobility_flags & MOBILITY_STAND))
 
 /mob/living/carbon/human/proc/fireman_carry(mob/living/carbon/target)
+	var/carryspeed = 0 //if you have latex you are faster at grabbing
+	if(HAS_TRAIT(src, TRAIT_QUICKER_CARRY))
+		carryspeed = 35
+	else if(HAS_TRAIT(src, TRAIT_QUICK_CARRY))
+		carryspeed = 40
+	else 
+		carryspeed = 50
 	if(can_be_firemanned(target))
-		visible_message("<span class='notice'>[src] starts lifting [target] onto their back...</span>",
+		if(carryspeed == 35)
+			visible_message("<span class='notice'>[src] starts expertly lifting [target] onto their back.</span>",
+			"<span class='notice'>Using your gloves' nanochips, you quickly start to lift [target] onto your back.</span>")
+		else if(carryspeed == 40)

```

Comment:
If it's only changed via trait, put it (being the difference between the two) there.

If it's more malleable having these at exactly 35/40 is odd